### ISSUE TITLE
repro: cwd vs wdir option inconsistancy

### ISSUE THREAD

**Issue by mroutis** [iterative/dvc#1739]
Does `dvc repro` still supports the `CWD` option?

```diff
Reproduce DVC file. Default file name - 'Dvcfile'.

positional arguments:
  targets               DVC file to reproduce.

optional arguments:
  -h, --help            show this help message and exit
  -q, --quiet           Be quiet.
  -v, --verbose         Be verbose.
  -f, --force           Reproduce even if dependencies were not changed.
  -s, --single-item     Reproduce only single data item without recursive
                        dependencies check.
+  -c CWD, --cwd CWD     Directory within your repo to reroduce from.
  -m, --metrics         Show metrics after reproduction.
  --dry                 Only print the commands that would be executed without
                        actually executing.
  -i, --interactive     Ask for confirmation before reproducing each stage.
  -p, --pipeline        Reproduce the whole pipeline that the specified stage
                        file belongs to.
  -P, --all-pipelines   Reproduce all pipelines in the repo.
  --ignore-build-cache  Reproduce all descendants of a changed stage even if
                        their direct dependencies didn't change.
  --no-commit           Don't put files/directories into cache.
```

**Comment by efiop** [comment]
@mroutis yes, it does. It is not the same as `dvc run -c` though, so it is not obsoleted :slightly_smiling_face:

**Comment by mroutis** [comment]
@efiop, got it, it is related to: https://github.com/iterative/dvc/issues/401

TLDR: same as `make -C`
